src/main/java/org/generationcp/middleware/pojos/oms/CVTerm.java

```diff
@@ -152,11 +153,10 @@ public void setDbxRefId(final Integer dbxRefId) {
 	}
 
 	public Boolean isObsolete() {
-		return this.isObsolete > 0;
+		return this.isObsolete != null && this.isObsolete > 0;
```

Comment:
Is there a reason why it needs to check if isObsolete is null? I think is_obsolete table is not null and defaults to 0 value.